druid-shell/src/platform/gtk/menu.rs

```diff
@@ -142,9 +143,16 @@ impl Menu {
 }
 
 fn register_accelerator(item: &GtkMenuItem, accel_group: &AccelGroup, menu_key: HotKey) {
-    let gdk_keyval = match menu_key.key {
-        KeyCompare::Code(key_code) => key_code.into(),
-        KeyCompare::Text(text) => text.chars().next().unwrap() as u32,
+    let gdk_keyval = match &menu_key.key {
+        Key::Character(text) => text.chars().next().unwrap() as u32,
```

Comment:
Appreciate this was existing code, but this unwrap feels bad, do we have a way of ensuring `Character` is never the empty string?

Reply:
Right, if I were defining the type I'd have nonempty as an invariant. But in this particular code path we're basically consuming client code that can construct the value directly. I'm comfortable making this an `expect` with a helpful message, as I think `Key::Character("")` is something that is (a) unlikely and (b) reasonable to expect to fail.